pkg/pillar/docs/radio-silence.md

```diff
@@ -30,34 +31,51 @@ Radio silence implementation is split between 3 EVE microservices:
   `ChangeRequestedAt` from `ZedAgentStatus.RadioSilence` to `DeviceNetworkStatus.RadioSilence`,
   sets `ChangeInProgress` to `true` and starts switching radios of wireless devices ON/OFF.
   For WiFi adapters, this is done by directly [calling the rfkill command](../devicenetwork/wlan.go).
-  For cellular modems, NIM updates the configuration file `/run/wwan/config.json`, which is picked up
-  by the `wwan` microservice, and waits for the status update published in `/run/wwan/status.json` (see below).
+  For cellular modems, NIM updates and publishes new `WwanConfig` via pubsub, which is received
+  by `mmagent` of the `wwan` microservice. NIM then waits for the status update from `wwan`
+  published through pubsub topic `WwanStatus`. To avoid NIM processing obsolete `WwanStatus`,
+  corresponding to an older `WwanConfig` and thus not reflecting the latest radio config, a timestamp
+  is appended to `WwanConfig` by NIM (as max of `ChangeRequestedAt` from `ZedAgentStatus.RadioSilence`
```

Comment:
Does this max assume tgat ChangesRequestedAt and  TimePriority assume that they are both synchronized clocks?
Wouldn'tt be more robust to have nim increment a version in the WwanConfig each time it sees a change in either ChangesRequestedAt or TimePriority?

Reply:
You are right, this is not a good approach. Additionally, NIM may fallback to an older DPC and in such case the wwanConfig timestamp might not change (if RadioSilence.ChangeRequestedAt is greater than the timestamp of the previous DPC).
I decided to simply put both DPC key+timestamp and RadioSilence timestamp into WwanConfig, instead of trying to determine the wwan config version somehow.
Having sources of `WwanConfig` identified also helps with debugging. For a given `WwanConfig`, it will be possible to go backwards in logs and trace the origins of the corresponding DPC and RadioSilence config that `WwanConfig` was generated from.
Documentation was updated accordingly.

Reply:
Thanks.